unit_tests/src/unit_test_utils.hpp

```diff
@@ -116,13 +115,13 @@ namespace unit_test_utils
         }
 
         for (std::size_t i = 0; i < _size_in_bytes; ++i) {
-            out << _ch;
+            out.write(&_ch, 1);
```

Comment:
just curious... why is it '1' here?

https://www.cplusplus.com/reference/ostream/ostream/write/

Reply:
We are writing one byte to the stream.

Reply:
oh, dang, a for loop.   good lord.   carry on.